### DISCUSSION TITLE
Where can I download the release cuda 1~4 compressed files?

### DISCUSSION THREAD

**Discussion by jasonsu123** [hacksider/Deep-Live-Cam#718] [General]
Hello everyone, I recently saw a video on YouTube sharing this amazing tool, but I couldn't find the release 1~4 compressed files mentioned in the video in the current releases. May I ask where I can still download the older versions?

Thank you!

**Comment by KRSHH** [comment]
Quick Installer is in Gumroad - https://hacksider.gumroad.com/l/vccdmm

**Reply by jasonsu123** [reply]
Thank you for the reply. So, is there no longer a free trial version of Deep-Live-Cam available? Does that mean I can only ask people who downloaded it before? The link you provided is for the paid version, right?
https://hacksider.gumroad.com/l/vccdmm
Thank you.

**Reply by KRSHH** [reply]
Deep-Live-Cam is completely free and opensource, you can follow the steps in the README to set it up it. the Installer which was present before was older version. and the latest installer is available on Gumroad, It has constant updates and support.

**Reply by hacksider** [reply]
There was no such thing as a trial version. You can always install it free if you follow the instructions on the readme.
